setup.py

```diff
@@ -12,11 +12,16 @@ def read(fname):
 install_requires = [
     'comtypes', 'enum34;python_version<"3.4"', 'psutil', 'future']
 setup(name='pycaw',
-      version='20190904',
+      version='20210501',
```

Comment:
Ouch I haven't released for all this time!
I would not bump this as part as the PR.
And that's a good reminder for me to make a release soon actually :sweat_smile:

Reply:
haha 😂

yeah of course- with all these brand new features 🤩

by the way: do you think we could just drop the python2 support?
I see a lot of big packages doing that... wouldn't it be time for that?

Reply:
haha 😂

yeah of course- with all these brand new features 🤩

by the way: do you think we could just drop the python2 support?
I see a lot of big packages doing that... wouldn't it be time for that?

Reply:
Yeah I've been wondering for Python 2 so I'm glad you ask.
Python 2 officially retired January last year I think https://pythonclock.org/
I think we can release this version to be the last to support Python 2 and then drop it straight after that.

Reply:
hmmm does this mean ALL the new features need to be py2 compatible 😅

Reply:
That's a good question, but new examples are already not Python2 compatible right?
I'll try to release this week or weekend, hold your horses :smile:

Reply:
Honestly i have no clue what works in Python 2 or not 😇 (I have never used it ^^)
... besides the obvious like print() and f strings.

So i guess its not quite 100% compatible...

I would be in for #team_lazy and drop the py2 support now 😂 its 2021!
We could leave the legacy release on github or something...